src/library_dylink.js

```diff
@@ -554,12 +560,15 @@ var LibraryDylink = {
   // promise that resolves to its exports if the loadAsync flag is set.
   $loadWebAssemblyModule__docs: '/** @param {number=} handle */',
   $loadWebAssemblyModule__deps: [
-    '$loadDynamicLibrary', '$createInvokeFunction', '$getMemory',
+    '$loadDynamicLibrary', '$getMemory',
     '$relocateExports', '$resolveGlobalSymbol', '$GOTHandler',
     '$getDylinkMetadata', '$alignMemory', '$zeroMemory',
     '$alignMemory', '$zeroMemory',
     '$CurrentModuleWeakSymbols', '$alignMemory', '$zeroMemory',
-    '$updateTableMap',
+    '$updateTableMap'
+#if !DISABLE_EXCEPTION_CATCHING || SUPPORT_LONGJMP == 'emscripten'
+     , '$createInvokeFunction'
```

Comment:
micro nit: I usually leave the trailing comma on the line before rather than leading with a comma here.. so we get nicer vertical alignment.

Also I like to end the list in a comma, so when new lines are added it only need to include changes on that new line.